webofneeds/won-owner-webapp/src/main/webapp/app/components/post-messages.js

```diff
@@ -226,9 +226,10 @@ function genComponentConf() {
 
             <chat-textfield-simple
                 placeholder="::'Message (optional)'"
-                on-submit="::self.sendRequest(value)"
+                on-submit="::self.sendRequest(value, selectedPersona)"
                 allow-details="::false"
                 allow-empty-submit="::true"
+                show-personas="::true"
```

Comment:
shouldnt we only display personas if the user has them?

Reply:
This is just a flag that's used to signify that this button can have a persona selection, the actual selection is only visible if the user is also logged in. 

It would be possible to also hide the selection if the logged-in user has no personas, but I think in this case it would be better to display an add persona button or a message that there are no personas yet.

Reply:
ok